plugins/cisco_firepower_management_center/icon_cisco_firepower_management_center/util/utils.py

```diff
@@ -0,0 +1,30 @@
+from .scan_results import ScanResults
+from insightconnect_plugin_runtime.exceptions import PluginException
+import validators
+
+
+# Return the first element of an array, otherwise None.
+def first(array):
+    return next(iter(array or []), None)
+
+
+def generate_payload(results, operation, max_page_size):
+    scan_results = ScanResults(max_page_size)
+
+    for result in results:
+        host = result.get("host", {})
+        ip_address = host.get("ip_address", "")
+        if ip_address:
```

Comment:
You could also eliminate line 15 and 16 and make something like that 
```suggestion
        if ip_address := result.get("host", {}).get("ip_address", ""):
```

Reply:
I can't because I use the `ip_address` variable below.